lib/Runtime/PlatformAgnostic/Platform/POSIX/UnicodeText.cpp

```diff
@@ -46,7 +46,8 @@ namespace PlatformAgnostic
             return true;
         }
 
-        int32 ChangeStringLinguisticCase(CaseFlags caseFlags, const char16* sourceString, uint32 sourceLength, char16* destString, uint32 destLength, ApiError* pErrorOut)
+        template<bool toUpper, bool useInvariant>
+        charcount_t ChangeStringLinguisticCase(const char16* sourceString, charcount_t sourceLength, char16* destString, charcount_t destLength, ApiError* pErrorOut)
```

Comment:
>Linguistic [](start = 32, length = 10)

how do we handle the langtag for this?

Reply:
POSIX doesn't have any idea of locale for this implementation, but this is only used in the --no-icu case, which as far as I know is anything goes in terms of spec compliance. This didn't handle system vs root locale before, either.